### PR TITLE
add parquet-schema

### PR DESCRIPTION
**Pull request by shannonwells**
Problem
=======
To write batch files we will need Parquet schema for every DSNP Announcement type.
[Finishes #177940101](https://www.pivotaltracker.com/story/show/177940101)

TODO: review the schema and make sure they are up to date with the message fields.

Solution
========
Add the schema files

with @pairperson1

### CODE REVIEW

**Comment by enddynayn** [comment]
looking great!

**Comment by enddynayn** [comment]
amazing!

**Review by enddynayn — APPROVED** [review]

File: src/batch/parquet-schema.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * Broadcast: a public post
+ */
+export const BroadcastSchema = {
+  actionType: { type: "INT32" },
+  fromAddress: { type: "BYTE_ARRAY" },
+  messageID: { type: "BYTE_ARRAY" },
+  uri: { type: "BYTE_ARRAY" },
+};
+
+export const ReplySchema = {
```
Comment (aneyzberg) [inline src/batch/parquet-schema.ts:16]: the other schemas have comments explaining what they are for. Do we want a comment for reply schema and drop schema below?

**Review by aneyzberg — APPROVED** [review]
small comment but otherwise lgtm

File: src/batch/parquet-schema.ts (hunk 1 above)
Comment (shannonwells) [reply src/batch/parquet-schema.ts:19]: Yes, I added them.

**Pull request merged**